website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -1,69 +1,673 @@
+
+Data type: string
+Default: none
+Required: Required for Kubernetes; not applicable for other runtimes 
+
+### Kind
+Specifies the configuration entry type.
+Data type: string 
+Default: none
+Required: required
+
+#### Virtual machines
+The value must be `service-defaults`. 
+
+#### Kubernetes
+Use lowercase `kind` in YAML. The value must be `ServiceDefaults`.
+
+
+### Name 
+Specifies the name of the service you are setting the defaults for. 
+
+Data type: string
+Default: none
+Required: required
+
+#### Kubernetes
+Specify the service name in the `metadata.name` field in YAML for Kubernetes. 
+
+### Namespace <Enterprise/>
+Specifies the Consul namespace that the configuration entry applies to. 
+
+Data type: string
+Default: `default`
+Required: optional
+
+#### Kubernetes
+Specify the namespace in the `metadata.namespace` field in YAML for Kubernetes. Refer to [Consul Enterprise](/consul/docs/k8s/crds#consul-enterprise) for information about how Consul namespaces map to Kubernetes Namespaces. Open source Consul distributions (Consul OSS) ignore the `namespace` configuration.   
+
+### Partition <Enterprise/>
+Specifies the name of the name of the Consul admin partition that the configuration entry applies to.  Refer to [Admin Partitions](/consul/docs/enterprise/admin-partitions) for additional information.
+
+Data type: string
+Default: `default`
+Required: optional
+#### Kubernetes
+Specify the admin partition in the `metadata.partition` field in YAML for Kubernetes. Consul OSS distributions ignore the `partition` configuration.   
+### Meta
+Specifies a set of custom key-value pairs. The field supports string values and stores them in the [Consul KV](/consul/docs/dynamic-app-config/kv). This field is not supported on Kubernetes.
+
+Data type: Object containing string key-value pairs
+Default: none
+Required: optional
+
+### metadata
+
+Kubernetes-only object that contains the service name, namespace, and admin partition that the configuration entry applies to. Refer to the following sections for configuration details:
+
+[`Name`](#name)
+[`Namespace`](#namespace)
+[`Partition`](#partition)
 
-Set default connection limits and mesh gateway mode across all upstreams
-of "counting", and also override the mesh gateway mode used when dialing
-the "dashboard" service.
+### spec
+Kubernetes-only object that contains the details about the `service-defaults` configuration entry. The `apiVersion`, `kind`, and `metadata` fields are siblings of the `spec` field. All other configurations are children.   
+
+### Protocol
+Specifies the default protocol for the service. This is used by Connect proxies for things like observability features and to unlock usage of the [\service-splitter`](/docs/connect/config-entries/service-splitter) and `service-router` config entries for a service. It also unlocks the ability to define L7 intentions via `service-intentions`.
```

Comment:
```suggestion
Specifies the default protocol for the service. This is used by Connect proxies for things like observability features and to unlock usage of the [\service-splitter`](/consul/docs/connect/config-entries/service-splitter) and `service-router` config entries for a service. It also unlocks the ability to define L7 intentions via `service-intentions`.
```
Hi 👋🏻 Now that the [link migration](https://github.com/hashicorp/consul/pull/15976) been released, I'm running through open PRs to help update legacy links. This should be prefixed with the product slug now! Let me know if you have questions.